include/oneapi/dpl/pstl/hetero/dpcpp/utils_ranges_sycl.h

```diff
@@ -162,7 +162,7 @@ template <typename _Range, typename... _Ranges>
 constexpr _Range
 __get_first_range(_Range&& __rng, _Ranges&&...)
 {
-    return __rng;
+    return ::std::forward<_Range>(__rng);
```

Comment:
I just figured out that we need to carefully think what's going on here. What this function return depends on deduced `Range` type that can be either `Range` or `Range&`